src/modules/design/property-panel/indextabs/general/sections/basics/basics.ts

```diff
@@ -239,14 +239,16 @@ export class BasicsSection implements ISection {
     return elementsWithSameId.length === 0;
   }
 
-  private isProcessIdUnique(id: string): boolean {
+  private areRootElementIdsUnique(id: string): boolean {
     // eslint-disable-next-line no-underscore-dangle
     const elementIds: Array<string> = this.modeler._definitions.rootElements.map((rootElement: IModdleElement) => {
       return rootElement.id;
     });
 
-    const currentId: number = elementIds.indexOf(id);
-    elementIds.splice(currentId, 1);
+    const currentId: number = elementIds.indexOf(this.businessObjInPanel.id);
```

Comment:
Man sollte die Variable als "Index" kennzeichnen, damit nicht der Eindruck entsteht, man hätte hier eine tatsächliche ID in der Hand.

```suggestion
    const currentElementIdIndex = elementIds.indexOf(this.businessObjInPanel.id);
```

Reply:
Stimmmt habe ich übersehen 👍